cli/src/main.rs

```diff
@@ -216,12 +216,19 @@ fn override_configuration(mut config: Config, options: &ArgMatches) -> Config {
     }
 
     #[cfg(feature = "go")]
-    if let Some(go_package) = options.value_of(args::ARG_GO_PACKAGE) {
-        config.go.package = go_package.to_string();
+    {
+        match options.value_of(args::ARG_GO_PACKAGE) {
+            Some(go_package) => {
+                config.go.package = go_package.to_string();
+            }
```

Comment:
since we would do nothing in the None branch this might be better expressed as an `if let` statement

e.g.
```rust
if let Some(go_package) = options.value_of(args::ARG_GO_PACKAGE) {
    config.go.package = go_package.to_string();
}
```